internal/server/handler.go

```diff
@@ -35,8 +37,15 @@ func (s *Server) handler() http.HandlerFunc {
 			s.metrics.observeRequest(r.Method, installationID, statusCode, elapsed)
 		}()
 
-		if s := strings.Split(r.URL.Path, "/"); len(s) > 1 {
-			installationID = s[1]
+		if s := strings.Split(r.URL.Path, "/"); len(s) > 2 {
+			installationID = s[2]
```

Comment:
Did the logic change? The bucket path prefix was the installation id right?

Reply:
I don't believe anything has changed. The path is something like `/BUCKET_NAME/INSTALLATION_ID/etc`

Reply:
Oh so we were doing it wrong from the beginning?